cirq/ops/controlled_gate.py

```diff
@@ -107,11 +161,17 @@ def _circuit_diagram_info_(self,
         if sub_info is None:
             return NotImplemented
         return protocols.CircuitDiagramInfo(
-            wire_symbols=('@',) + sub_info.wire_symbols,
+            wire_symbols=('@',)*self.num_controls() + sub_info.wire_symbols,
             exponent=sub_info.exponent)
 
     def __str__(self):
-        return 'C' + str(self.sub_gate)
+        return 'C'*self.num_controls() + str(self.sub_gate)
 
     def __repr__(self):
-        return 'cirq.ControlledGate(sub_gate={!r})'.format(self.sub_gate)
+        if (self.num_controls() is 1 and self.control_qubits[0] is None):
```

Comment:
Do not use `is` when comparing integers. I don't think it's guaranteed that every 1 is at the same location in memory. https://stackoverflow.com/questions/306313/is-operator-behaves-unexpectedly-with-integers